src/Microsoft.AspNetCore.Mvc.Core/ControllerContext.cs

```diff
@@ -56,56 +52,6 @@ public new ControllerActionDescriptor ActionDescriptor
         }
 
         /// <summary>
-        /// Gets or sets the list of <see cref="IInputFormatter"/> instances for the current request.
-        /// </summary>
-        public virtual FormatterCollection<IInputFormatter> InputFormatters
-        {
-            get
-            {
-                if (_inputFormatters == null)
-                {
-                    _inputFormatters = new FormatterCollection<IInputFormatter>();
-                }
-
-                return _inputFormatters;
-            }
-            set
-            {
-                if (value == null)
-                {
-                    throw new ArgumentNullException(nameof(value));
-                }
-
-                _inputFormatters = value;
-            }
-        }
-
-        /// <summary>
-        /// Gets or sets the list of <see cref="IModelValidatorProvider"/> instances for the current request.
-        /// </summary>
-        public virtual IList<IModelValidatorProvider> ValidatorProviders
```

Comment:
Wow, we were carting these two properties around for `TryUpdateModelAsync()` calls and that's it? No real scenario for controller code to change `ValidatorProviders` before calling `TryValidateModel()`?